pkg/scaling/scaledjob/metrics.go

```diff
@@ -0,0 +1,121 @@
+package scaledjob
+
+import (
+	"math"
+
+	v2 "k8s.io/api/autoscaling/v2"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+// GetTargetAverageValue returns the average of all the metrics' average value.
+func GetTargetAverageValue(metricSpecs []v2.MetricSpec) float64 {
+	var totalAverageValue float64
+	var metricValue float64
+	for _, metric := range metricSpecs {
+		if metric.External.Target.AverageValue == nil {
+			metricValue = 0
+		} else {
+			metricValue = metric.External.Target.AverageValue.AsApproximateFloat64()
+		}
+
+		totalAverageValue += metricValue
+	}
+	count := float64(len(metricSpecs))
+	if count != 0 {
+		return totalAverageValue / count
+	}
+	return 0
+}
+
+// CreateMetricSpec creates MetricSpec for given metric name and target value.
+func CreateMetricSpec(averageValue int64, metricName string) v2.MetricSpec {
```

Comment:
Isn't this used only in the test? Then it doesn't make sense to define here imho.

Reply:
I really appreciate your comment.
BTW, this function is used at scaler_handler_test.go as well., so I left here.
Could you give any suggestion?

Reply:
So would be better to move to the test file and make it private?

Reply:
Hi, what I mean is,
it's been used two different package(scaledjob/metric_test.go, scaling/scaler_handler_test.go). If i make it private, I need to write duplicated code at both. 
Any good convention for this case?

Reply:
That's right. I think that we can duplicate the function in this case. Moving it to some another package would bring to much complexity.

Reply:
That sounds much reasonable.
I updated it.
Thanks for the detail.